### DISCUSSION TITLE
Please add an option to share a specific line

### DISCUSSION THREAD

**Discussion by mrexodia** [community/community#8086] [Mobile]
For me this is a reason to constantly have to go back and forth between Safari and the app.

I want to share a link in a specific commit on Discord, I find my project on Google, click it which opens the GitHub app, then I have to copy the link and manually open it in the browser, click the line, click the address bar and copy the link to share.

I think if the line numbers were a bit wider clicking them could work like the web interface and then pressing "share" would include the link to the line.

**Comment by Vannsansan** [comment]
Hi

**Comment by sethfri** [comment]
I agree, this is a really frustrating part of the GitHub iOS app today. I need to share links to lines in files all the time with coworkers, but today I have to be at my desktop to do it

**Comment by haldun** [comment]
Hi @mrexodia, thanks for the feedback on this. I've filed a request and shared with the team for prioritization.

**Comment by leehagoodjames** [comment]
Please add support for this. I just figured out that this is not supported, and it's something that I use GitHub for all the time and really wish it was supported in the iOS mobile app
